src/api/uploads/__tests__/MultiputUpload.test.js

```diff
@@ -120,6 +120,46 @@ describe('api/uploads/MultiputUpload', () => {
             expect(multiputUploadTest.numPartsUploading).toBe(1);
             expect(multiputUploadTest.parts[0].upload).toHaveBeenCalled();
         });
+
+        test('should call unpause when part is paused and in digest ready state', () => {
```

Comment:
You could also use `test.each` for these tests

Reply:
I was not able to pass in the different function calls to `test.each` so left it as is.